eos-live-boot-overlayfs-setup

```diff
@@ -17,6 +17,14 @@ find_storage_partition() {
 	local root_partition=$(blkid --output device --match-token "${bootdev}")
 	[ -b "${root_partition}" ] || return 1
 
+	udevadm settle
+
+	eos-map-image-file ${root_partition} /endless/live_storage endless-live_storage
+	if [ $? = 0 ]; then
```

Comment:
Rather than doing this you could use:

```
if [[ -f "$PERSISTENT_IMG_PATH" ]] && eos-map-image-file …; then
```

Reply:
I'd rather make the udevadm settle conditional on the file existence too anyway, since I've seen
other commits where we go out of our way to avoid it (e4642cce73)